lib/holdings/status/deliver_from_offsite.rb

```diff
@@ -1,23 +1,23 @@
 class Holdings
   class Status
-    class Pageable
+    class DeliverFromOffsite
       def initialize(item)
         @item = item
       end
 
-      def pageable?
-        pageable_home_location? || pageable_library?
+      def deliver_from_offsite?
+        pageable_home_location? || offsite_library?
```

Comment:
Should `pageable_home_location?` and `Constants::PAGE_LOCS` (below) also be renamed? I'd guess something like `offsite_home_location?` given that they also get the truck icon, but I don't know where these locations are.

Reply:
I think that *could* be something we do, but I didn't really understand that part, so I scoped the changes to avoid that part. Since that's a private method, I'm not too concerned about it leading people astray.

Reply:
@corylown I changed it to `deliverable_home_location?` Does that work for you?